R/calculateDescriptives.R

```diff
@@ -97,16 +94,16 @@ varStats <- function(GADSdat, sub.inputForDescriptives, verbose, showCallOnly =
          }
      }  else  {
 ### differenzieren, ob es skala (es gibt eine separate skalenvariale) oder fake.skala (es gibt keine separate skalenvariale) ist
-         if ( "scale" %in% sub.inputForDescriptives[,"type"] ) {
-             if(length(which("scale" == sub.inputForDescriptives[,"type"])) != 1) {cat("Error: Activate browser.\n"); browser()}
+#         if ( "scale" %in% sub.inputForDescriptives[,"type"] ) {
+             if(length(which("scale" == sub.inputForDescriptives[,"type"])) > 1) {cat("Error: Activate browser.\n"); browser()}

```

Comment:
Nit: Could this be removed or turned into an informative error message?

Reply:
I would like to leave the browser in because otherwise the error will show up later. Also, it's a good indicator to see if something went wrong with the identification of actual or fake scales.